### PR TITLE
fix walletcreatefundedpsbt deriv paths, add test

### PR DESCRIPTION
**Pull request by instagibbs**
Added the regression in #13968

### CODE REVIEW

File: test/functional/rpc_psbt.py [hunk 1]
```diff
@@ -143,17 +143,19 @@ def run_test(self):
         # replaceable arg
         block_height = self.nodes[0].getblockcount()
         unspent = self.nodes[0].listunspent()[0]
-        psbtx_info = self.nodes[0].walletcreatefundedpsbt([{"txid":unspent["txid"], "vout":unspent["vout"]}], [{self.nodes[2].getnewaddress():unspent["amount"]+1}], block_height+2, {"replaceable":True})
+        psbtx_info = self.nodes[0].walletcreatefundedpsbt([{"txid":unspent["txid"], "vout":unspent["vout"]}], [{self.nodes[2].getnewaddress():unspent["amount"]+1}], block_height+2, {"replaceable":True}, False)
```
Comment (Empact) [inline test/functional/rpc_psbt.py]: nit: a comment for the boolean args would help with readability

**Comment by Empact** [comment]
utACK 61fe653

**Comment by achow101** [comment]
utACK 61fe653bd919cb0533b2b9d6259bc86a4b2975c0

**Comment by instagibbs** [comment]
possible backport for 0.17, in case we're doing another RC? @fanquake

**Pull request merged**